containers/flatpak/test/test-ssh

```diff
@@ -12,6 +12,7 @@ FLATPAK_ID="${1:-org.cockpit_project.CockpitClient}"
 if mkdir ~/.ssh; then
     # ~/.ssh didn't exist yet.  Assume that we're safe to configure this.
     echo 'Include cockpit-client-test.config' > ~/.ssh/config
+    trap 'rm -rf ~/.ssh' EXIT INT QUIT PIPE
```

Comment:
Why doesn't this work with the second branch which checks that this has been configured?

Reply:
Before I added that, [tests failed](https://github.com/cockpit-project/cockpit/runs/5596989327?check_suite_focus=true) the second time. That might be coincidental (I didn't investigate too much). But mostly, I find it cleaner in general: If the test creates a file, it should remove it again. For running this locally, I need to `mv ~/.ssh ~/.ssh.bak`, and renaming it back doesn't work as the directory already exists.

Reply:
You could also just add the line to your ssh config :)